rfc_process.md

```diff
@@ -0,0 +1,83 @@
+* All expected behavior should be documented, in sufficient detail such that language-specific tests and examples can be generated from the descriptions.
+* Risks, backwards compatibility, and potential for inconsistency are addressed.
+
+## Tracking Issue
+Often, a proposal will consist of multiple artifacts, such as GitHub issues, pull requests, release candidates, etc. In order to manage these resources, a single, long-running tracking issue is assigned to each proposal.
+
+A tracking issue consists of the following:
+
+* Status, date, and author.
+* Short description of the proposal.
+* A link to the committed RFC.
+* Lists of open issues, PRs, and upcoming deadlines.
+* Links to relevant artifacts, such as release candidates.
+* Links to meetings and other discussion channels.
+
+## Release Candidates
+Each language interface implements the abstract interface contained in the RFC.
+
+A complete release candidate consists of the following:
+* Repository branch containing the new version.
+* Packages for installing the new version.
+* Tracers which bind to the new version.
+* Major instrmentation ported to the new version.
+
+# Proposal Lifecycle
+
+## Draft
+* Pull request to add the draft RFC to `/rfc`. 
+* Tracking issue for the proposal is opened.
+* Debate continues via pull requests against the draft.
+
+A draft proposal is first submitted as a pull request against the specification repository. 
+
+Once there is consensus that the pull request meets the minimum criteria for the beginnings of a draft, and has strong potential to be accepted, the pull request is merged. A tracking issue is created at this point to manage the proposal. The draft RFC is then refined via more pull requests.
+
+Consensus is used to determine that all of the abstract proposal work has been completed, and the investigation should move to language implementations.
+
+## Test
+* RFC status changed from `Draft` to `Test`.
+* Release Candidate per major language is created.
+* API conventions are coordinated between languages.
+* New risks and ambiguities may lead to proposal changes.
+
+Once the draft proposal is finalized, language implementation begins. Language maintainers begin creating a release candidate on a branch of the language repository. Links to the release candidates, along with their status, are recorded in the tracking issue.
+
+## Accepted
+* Remaining language interfaces are released.
+* Abstract interface is versioned and updated.
+* Documentation is updated.
+* RFC status changed from `Test` to `Accepted`.
+* Tracking issue for the proposal is closed.
+
+After a quorum of language interfaces have been tested and released, consensus may be reached that a proposal has been finalized. At this point, the abstract interface contained in the `specification.md` document is updated to reflect the final changes, based on the language in the RFC docment and release candidates.
```

Comment:
Isn't this already done via the "Accepted" stage that says that the "Abstract interface is versioned and updated"?

Reply:
Not sure what distinction you are making. This is a description of the Accepted stage.

Reply:
my bad - I thought this was a new paragraph.